edumap/app/models/lesson.rb

```diff
@@ -10,61 +10,48 @@ class Lesson < ActiveRecord::Base
 
   self.per_page = 10
 
-  # belongs_to :unit
   belongs_to :curriculum
   has_and_belongs_to_many :codes
   has_and_belongs_to_many :levels
   has_and_belongs_to_many :standards
 
   scope :search_query, lambda { |query|
-    return nil  if query.blank?
-    # condition query, parse into individual keywords
+    return nil if query.blank?
+    # most people use * for wildcard searches
+    # MySQL uses % instead for reasons
     terms = query.downcase.split(/\s+/)
-    # replace "*" with "%" for wildcard searches,
-    # append '%', remove duplicate '%'s
-    terms = terms.map { |e|
-      ("%" + e.gsub('*', '%') + '%').gsub(/%+/, '%')
-    }
-    # configure number of OR conditions for provision
-    # of interpolation arguments. Adjust this if you
-    # change the number of OR conditions.
-    num_or_conditions = 2
-    where(
-      terms.map {
-        or_clauses = [
-          "LOWER(lessons.name) LIKE ?",
-          "LOWER(lessons.lesson_url) LIKE ?",
-        ].join(' OR ')
-        "(#{ or_clauses })"
-      }.join(' AND '),
-      *terms.map { |e| [e] * num_or_conditions }.flatten
-    )
+                 .map { |e| "%#{e.gsub('*', '%')}%" }
```

Comment:
Honestly this should probably be split into `.map {|e| e.gsub /*/, /%/}.map{|e| "%#{e}%"}` to make it a bit clearer that it's two different processings